doc/api.rst

```diff
@@ -12,8 +12,15 @@ Here we list the Application Programming Interface (API) for MNE-BIDS.
    :local:
    :depth: 2
 
-MNE BIDS (:py:mod:`mne_bids`)
-=============================
+
+MNE BIDS
+========
+
+:py:mod:`mne_bids`:
+
+.. automodule:: mne_bids
+   :no-members:
+   :no-inherited-members:
```

Comment:
There previously was no link target to `mne_bids` (there was what appeared to be an attempt to add one but it was incorrect/not working). Doing `automodule` will create this target properly, and doing `:no-members:` will make it so that you get to manually list the objects to document